src/pages/likedVideoPage/likedVideoPage.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
```

Comment:
changes: the file name should be LikedVideoPage.js instead of likedVideoPage.js because it is a react component.